### ISSUE TITLE
"Latest published version" in editor's draft points to wrong place

### ISSUE THREAD

**Issue by jyasskin** [speced/respec#4688]
In https://w3c.github.io/geolocation-api/, the "Latest published version:" points to https://www.w3.org/geolocation/, which is not a version of this spec. I think it should point to https://www.w3.org/TR/geolocation/.

**Comment by marcoscaceres** [comment]
I'll take a look at what's going on... For whatever reason, respec is not producing the right URL.

**Comment by floresbakker** [comment]
I have a similar thing for my community group htmlvoc.  Respec derives correctly that the "Latest published version:" is
"https://github.com/floresbakker/htmlvoc" but for some reason comes up with a non-existent URL for "Latest editor's draft:": "https://floresbakker.github.io/htmlvoc/". Where does it get this information from? My W3C profile does not have this URL, nor does the page of my community group at https://www.w3.org/community/htmlvoc/

This is the ReSpec configuration:


var respecConfig = {
--
  | specStatus: "CG-DRAFT",
  | latestVersion: "https://github.com/floresbakker/htmlvoc",
  | editors: [{name: "Flores Bakker"},{name: "Thomas de Groot"},{name: "Wouter Beek"},],
  | github: "https://github.com/floresbakker/htmlvoc",
  | shortName: "htmlvoc",
  |  
  | group: "htmlvoc",
  | };

**Comment by marcoscaceres** [comment]
This is a spec-level config issue. The `latestVersion` URL is derived from `pubSpace/shortName`, which produces the correct `/TR/` URL. If the URL is wrong, the spec's `latestVersion` or `shortName` config needs adjustment. Not a ReSpec code bug.

**Comment by marcoscaceres** [comment]
Reopening: closed prematurely without proper verification. Will re-investigate with /ai-skeptic validation before re-closing.

**Comment by marcoscaceres** [comment]
Verified: the spec's config now has `latestVersion: "https://www.w3.org/TR/geolocation/"` (correct /TR/ URL). The issue was in the spec's config, not ReSpec code. Confirmed by fetching the current index.html from w3c/geolocation-api.

**Comment by jyasskin** [comment]
It seems like a workaround to make specs explicitly list their published version, instead of having respec figure it out, especially when the correct published URL is in TR space. Does your AI prompt check whether the change to the spec's config is supported in respec documentation?

**Comment by marcoscaceres** [comment]
Reopening. Jeffrey correctly points out that the spec shouldn't need to explicitly set `latestVersion` when it's a standard /TR/ spec. ReSpec should derive the correct URL from `pubSpace/shortName`. Will investigate why the auto-generation produced the wrong URL for the original reporter.

**Comment by marcoscaceres** [comment]
Yeah, the AI was being overly aggressive... I told it to fix 50 issues in one go and it got a bit too eager about hitting the number and didn't do sufficient checks. I sat it down and gave it a talking to. Adjusting the strategy. Thanks for the feedback, @jyasskin!
